examples/ecs/ecs_guide.rs

```diff
@@ -238,7 +238,7 @@ fn print_at_end_round(mut counter: Local<u32>) {
     println!();
 }
 
-#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+#[derive(StageLabel)]
```

Comment:
I take it the other traits are implicitly implemented now?

Reply:
No, they're just no longer necessary for labels. And since this is an example, removing the extra derives won't break anyone.